screens/HotspotDetailScreen.tsx

```diff
@@ -0,0 +1,281 @@
+    },
+    separator: {
+      borderColor: theme.colors.disabled,
+      borderWidth: 0.7,
+      borderLeftWidth: 0,
+      borderRightWidth: 0,
+      height: 4,
+      marginBottom: 16,
+      marginTop: 16,
+    },
+    addressTitle: {
+      fontSize: 18,
+      fontFamily: 'Nunito_700Bold',
+      letterSpacing: 0.2,
+      textAlign: 'left',
+    },
+    subTitle: {
+      fontFamily: 'Nunito_400Regular',
+      fontSize: 15,
+      color: theme.colors.placeholder,
+      paddingTop: 11,
+      paddingBottom: 8,
+    },
+    button: {
+      width: '100%',
+    },
+    buttonContent: {
+      alignItems: 'center',
+      justifyContent: 'center',
+      flexDirection: 'row-reverse',
+      padding: 8,
+    },
+    buttonLabel: {
+      color: theme.colors.background,
+      fontSize: 16,
+      fontFamily: 'Nunito_700Bold',
+    },
+    notes: {
+      paddingTop: 12,
+      color: theme.colors.placeholder,
+      fontFamily: 'Nunito_400Regular',
+      fontSize: 16,
+      marginBottom: 24,
+    },
+    image: {
+      width: '100%',
+      maxHeight: scaledHeight,
+      position: 'absolute',
+      bottom: 0,
+    },
+    catCategoryContainer: {
+      flexDirection: 'row',
+      alignItems: 'center',
+      marginTop: 12,
+      marginBottom: 12,
+    },
+    catCategoryTitleIcon: {
+      backgroundColor: theme.colors.text,
+    },
+    catCategoryTitleLabel: {
+      fontSize: 18,
+      fontFamily: 'Nunito_700Bold',
+      color: theme.colors.text,
+      marginStart: 8,
+    },
+    scrollView: {
+      width: '100%',
+      height: '1000%',
+    },
+    statusView: {
+      paddingTop: 14,
+      paddingBottom: 14,
+      justifyContent: 'center',
+      alignItems: 'center',
+      borderRadius: theme.roundness,
+      height: 100,
+      flex: 1,
+      margin: 5,
+      marginLeft: 0,
+    },
+    statusText: {
+      color: theme.colors.text,
+      fontSize: 12,
+      fontFamily: 'Nunito_700Bold',
+      textTransform: 'uppercase',
+      textAlign: 'center',
+      flexWrap: 'wrap',
+    },
+    informationView: {
+      paddingTop: 14,
+      paddingBottom: 14,
+      backgroundColor: theme.colors.onSurface,
+      alignItems: 'center',
+      borderRadius: theme.roundness,
+      height: 100,
+      flex: 1,
+      margin: 5,
+      justifyContent: 'space-between',
+      flexDirection: 'column',
+    },
+    informationTitle: {
+      color: theme.colors.notification,
+      fontSize: 12,
+      fontFamily: 'Nunito_700Bold',
+      textAlign: 'center',
+      flexWrap: 'wrap',
+      textTransform: 'uppercase',
+    },
+    informationSubtitle: {
+      color: theme.colors.text,
+      fontSize: 14,
+      fontFamily: 'Nunito_700Bold',
+      textAlign: 'center',
+    },
+  });
+
+export const HotspotDetailScreen = ({
+  route,
+}: NativeStackScreenProps<RootStackParamList, 'HotspotDetail'>) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+  const hotspot = route.params.hotspot;
```

Comment:
By looking at the server api, I think we can't use this.
From the server we can load either a list of hotspots with light data for each one (id, lat, long, status) or
each hotspot by `id` with all the details (address, volunteer, cats...)

So what we have to do here is pass the hotspot id via route params and use an make a request to load all the details for the hotspot.

You can use my PR as an example.

We can do this in another PR.

Reply:
For now I leave this at it is, and do the changes in a separate branch.

Reply:
I'll in a new branch